Bruin Bite/View Controllers/PopupViewControllers/TimePickerViewController.swift

```diff
@@ -67,7 +67,7 @@ class TimePickerViewController: UIViewController {
             times.append(time)
         }
     } else {
-        print("No meal period selected")
+        Logger.shared.handle(type: .warning, message: "No meal period selected")
```

Comment:
Confirm if this needs to be `.error` or can stay as a `.warning`

Reply:
This warning/error occurs when someone tries to select a time before selecting a meal period. The resulting behavior is simply a popup that is not populated with times. I would not consider it a fatal error by any means. I suggest keeping it a warning and later alerting the user to this error when they attempt to put in a time before the meal period.